DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -195,7 +197,8 @@ private static ServiceAccountCredential GetCredential(string path = "credentials
 
             return GoogleCredential.FromStream(stream)
                 .CreateScoped(scopes)
-                .UnderlyingCredential as ServiceAccountCredential;
+                .UnderlyingCredential as ServiceAccountCredential
+                ?? throw new EventsSheetsInitialisationException("Credential maker returned null");
```

Comment:
Should be indented? 🙂

Reply:
Yep, sorry about that! Fixed in cc12dca 🙂